Dockerfile

```diff
@@ -23,5 +23,4 @@ COPY README.md /home/sa_scanner
 WORKDIR /home/sa_scanner
```

Comment:
It's set here already @ZetaTwo

Reply:
Ok, then it makes sense. Your initial comment was quite confusing.